packages/@o3r/extractors/src/core/comparator/package-managers-extractors/yarn2-file-extractor.helper.ts

```diff
@@ -0,0 +1,194 @@
+import {
```

Comment:
Maybe you should disable linter for this file as it is a fork of an open source code?

BTW, do they provide a license allowing the copy without author mention?
Maybe the warning regarding the fact it is a copy should be on the first lines of this file

Reply:
created issue https://github.com/AmadeusITGroup/otter/issues/1980 to check globally the licenses from third party code